restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -887,12 +912,15 @@ private void updateExistenceForMap(
         ImmutableList.of(BuiltCondition.of("key", Predicate.IN, new ArrayList<>(docNames)));
 
     db = dbFactory.getDocDataStoreForToken(authToken);
+
     List<Row> rows = db.executeSelect(keyspace, collection, predicate).rows();
     Map<String, List<Row>> rowsByDoc = new HashMap<>();
     for (Row row : rows) {
       String key = row.getString("key");
       List<Row> rowsAtKey = rowsByDoc.getOrDefault(key, new ArrayList<>());
-      if (fields.isEmpty() || fields.contains(row.getString("p0"))) rowsAtKey.add(row);
```

Comment:
Did our style rules change? pretty sure `mvnw xml:format format` caused this `if` to end up this way, weird

Reply:
Maybe? Personally I'm all for it. I know java supports one line if without curly braces but I'm not a fan.